platform/android/src/native_map_view.cpp

```diff
@@ -1050,6 +1050,14 @@ jni::jboolean NativeMapView::getPrefetchTiles(JNIEnv&) {
     return jni::jboolean(map->getPrefetchZoomDelta() > 0);
 }
 
+void NativeMapView::setPrefetchZoomDelta(JNIEnv&, jni::jint delta) {
+    map->setPrefetchZoomDelta(uint8_t(abs(delta)));
```

Comment:
Since we expect a positive value and negative ones don't make much sense, I'd rather do nothing or if-check whether the value actually is positive (print a warning otherwise) rather than convert ourselves which might introduce surprising misbehaviors for developers that made a calculation error. Wdyt?

Reply:
Yes, make sense, I will add a negative check in the java code instead of doing a conversion here.